feature-group-definitions/basic-arrays.yml

```
@@ -0,0 +1,46 @@
+name: JavaScript arrays
```

```suggestion
name: Array
```
 Can you also name it `array.yml`?